src/_pytest/python.py

```diff
@@ -1550,14 +1563,13 @@ def _showfixtures_main(config: Config, session: Session) -> None:
         tw.write(f"{argname}", green=True)
         if fixturedef.scope != "function":
             tw.write(" [%s scope]" % fixturedef.scope, cyan=True)
-        tw.write(f" -- {bestrel}", yellow=True)
+        tw.write(f" -- {prettypath}", yellow=True)
         tw.write("\n")
-        loc = getlocation(fixturedef.func, str(curdir))
         doc = inspect.getdoc(fixturedef.func)
         if doc:
             write_docstring(tw, doc.split("\n\n")[0] if verbose <= 0 else doc)
         else:
-            tw.line(f"    {loc}: no docstring available", red=True)
+            tw.line(f"    no docstring available", red=True)
```

Comment:
removed the location here as it seems redundant now that we always show the location anyways.